cpp/examples/CMakeLists.txt

```diff
@@ -34,6 +34,10 @@ add_executable(sde_sirs_example sde_sirs.cpp)
 target_link_libraries(sde_sirs_example PRIVATE memilio sde_sirs)
 target_compile_options(sde_sirs_example PRIVATE ${MEMILIO_CXX_FLAGS_ENABLE_WARNING_ERRORS})
 
+add_executable(sde_seir2v_example sde_seir2v.cpp)

```

Comment:
This will get changed from seir2v to seirvv even though I am not sure if this is consistent with secirvvs. Why is there a s after the "vv"?

Reply:
Okay, the easy part first - there is a flow from I to S (omitting immunity levels), hence the trailing "s". 

But you are right, the "vv" naming includes a "vaccination" part, but don't ask which v that is... Maybe "seirv" would be more in line with the current naming, but since there do not appear to be any general conventions here, you can keep the "2v" naming, if you prefer it.

Reply:
I changed it to seirvv for now but this doesn't feel right after your explanation. Even in this case it is not "only" a seir model with two variants. There are some immunity assumptions in this model, i.e. after infection with the variant type you can't get infected with the wild type.

Maybe Breadth-first search would make sense but "seeiirreir" really is not a nice name and in bigger models it would get really large.

Reply:
If the name doesn't feel right just rename it again.
The most important part is probably that the model name is somewhat descriptive, but unique within memilio. Both "seir2v" and "seirv(v)" work in that regard. If you want to add the immunity part, maybe "seirvi" and explain what i and v are in the Readme?

Reply:
I will just keep seirvv for now because the two v are after the last compartment (recovered).